.vuepress/public/sample-template.yaml

```diff
@@ -17,6 +17,7 @@ delete_one_document_1: |-
 delete_documents_1: |-
 search_post_1: |-
 get_update_1: |-
+getting_started_get_update: |-
```

Comment:
Same comment as above in `.code-samples.meilisearch.yaml`. It helps us keep these files tidy if code samples from the same page or site section are grouped together.